### PR TITLE
[13.0][MIG] website_form

### PR DESCRIPTION
**Pull request by bishalgit**
Nothing to do in this module.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+NEW ir.ui.view: website_form.ir_model_fields_view
+NEW ir.ui.view: website_form.ir_model_view
+NEW ir.ui.view: website_form.register_s_website_form
+NEW ir.ui.view: website_form.remove_external_snippets
+NEW ir.ui.view: website_form.s_website_form
+NEW ir.ui.view: website_form.snippet_options
+# NOTHING TO DO: new xmls
+
+NEW website.page: website_form.contactus_thanks [renamed from website_crm module]
+# NOTHING TO DO: noupdate=0: moved from website_crm to website_form
```
Comment (MiquelRForgeFlow) [inline addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt:21]: No, here you need in pre-migration call the rename_xmlids method

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (MiquelRForgeFlow) [reply addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt:21]: sorry, I said nothing

Comment (pedrobaeza) [inline addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt]: This requires some work for not losing custom "Thank you" pages, due to the COW (Create On Write) system. You have to:

- Effectively rename XML-IDs on pre. Although they are noupdate=0, there are some references to the view ID on the COW code, so keeping the same ID will help. This also helps with subsequent inheritance of the regular views.
- Update through a query all the views with the same old `key` field to the new key.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (bishalgit) [reply addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt]: Done

**Comment by bishalgit** [comment]
@pedrobaeza I have made the changes. Can you review it now?

**Comment by pedrobaeza** [comment]
The query for renaming existing ir.ui.views with old `key` to the new one is not yet performed.

**Comment by bishalgit** [comment]
> The query for renaming existing ir.ui.views with old `key` to the new one is not yet performed.

So renaming is not enough? Then should I also rename `key` as well?

**Comment by pedrobaeza** [comment]
As said, there are extra views that are created called "COW views", that are different from the one you are renaming.

**Comment by pedrobaeza** [comment]
As I can't push to corporate branches, I supersede this in #2582 for adding what I'm talking about the key.

**Pull request closed without merging**